data/styles/clear/include/Icons.mapcss

```diff
@@ -2673,6 +2673,17 @@ node|z18-[amenity=parking],
 area|z18-[amenity=parking]
 {icon-image: parking-m.svg;font-size: 11;text-optional: true;}
 
+node|z17[amenity=parking_entrance],
+area|z17[amenity=parking_entrance]
+{icon-image: parking_entrance-m.svg;}
+node|z17[amenity=parking_entrance][access=permissive],
+node|z17[amenity=parking_entrance][access=private]
```

Comment:
"z17-" to void private/permissive icons in all zooms?

Reply:
I copied the same logic that is used just above for amenity=parking

Reply:
Seems like you forgot to make generate_styles.sh
After that, if you check drules_proto_clear.txt, amenity-parking_entrance-private will be drawable with icon in z18+

Reply:
Fixed. Thanks!

Reply:
Ok, seems like we should revise permissive/private logic here, but in the follow up PRs.